pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -96,6 +96,7 @@ elixir-editors/vim-elixir
 elmcast/elm-vim
 elzr/vim-json
 embear/vim-localvimrc
+embear/vim-localvimrc
```

Comment:
this file was already added, please drop commit
`git rebase -i` makes this easy